app/celery/letters_pdf_tasks.py

```diff
@@ -1,424 +0,0 @@
-    current_app.logger.info('notification id %s Virus scan passed: %s', notification.id, filename)
-
-    is_test_key = notification.key_type == KEY_TYPE_TEST
-
-    scan_pdf_object = s3.get_s3_object(current_app.config['LETTERS_SCAN_BUCKET_NAME'], filename)
-    old_pdf = scan_pdf_object.get()['Body'].read()
-
-    try:
-        billable_units = get_page_count(old_pdf)
-    except PdfReadError as e:
-        current_app.logger.exception(e)
-        current_app.logger.error('Invalid PDF received for notification_id: %s', notification.id)
-        _move_invalid_letter_and_update_status(notification, filename, scan_pdf_object)
-        return
-
-    sanitise_response = _sanitise_precompiled_pdf(self, notification, old_pdf)
-    if not sanitise_response:
-        new_pdf = None
-    else:
-        sanitise_response = sanitise_response.json()
-        try:
-            new_pdf = base64.b64decode(sanitise_response['file'].encode())
-        except JSONDecodeError:
-            new_pdf = sanitise_response.content
-
-        redaction_failed_message = sanitise_response.get('redaction_failed_message')
-        if redaction_failed_message and not is_test_key:
-            current_app.logger.info(
-                '%s for notification id %s (%s)', redaction_failed_message, notification.id, filename
-            )
-            copy_redaction_failed_pdf(filename)
-
-    # TODO: Remove this once CYSP update their template to not cross over the margins
-    if notification.service_id == UUID('fe44178f-3b45-4625-9f85-2264a36dd9ec'):  # CYSP
-        # Check your state pension submit letters with good addresses and notify tags, so just use their supplied pdf
-        new_pdf = old_pdf
-
-    if not new_pdf:
-        current_app.logger.info('Invalid precompiled pdf received %s (%s)', notification.id, filename)
-        _move_invalid_letter_and_update_status(notification, filename, scan_pdf_object)
-        return
-    else:
-        current_app.logger.info('Validation was successful for precompiled pdf %s (%s)', notification.id, filename)
-
-    current_app.logger.info('notification id %s (%s) sanitised and ready to send', notification.id, filename)
-
-    try:
-        _upload_pdf_to_test_or_live_pdf_bucket(new_pdf, filename, is_test_letter=is_test_key)
-
-        update_letter_pdf_status(
-            reference=reference,
-            status=NOTIFICATION_DELIVERED if is_test_key else NOTIFICATION_CREATED,
-            billable_units=billable_units,
-        )
-        scan_pdf_object.delete()
-    except BotoClientError:
-        current_app.logger.exception('Error uploading letter to live pdf bucket for notification: %s', notification.id)
-        update_notification_status_by_id(
-            notification.id, NOTIFICATION_TECHNICAL_FAILURE, status_reason='Internal server error'
-        )
-
-
-def _move_invalid_letter_and_update_status(
-    notification,
-    filename,
-    scan_pdf_object,
-):
-    try:
-        move_scan_to_invalid_pdf_bucket(filename)
```

Comment:
It appears `move_scan_to_invalid_pdf_bucket` can be removed from the app.letters.utils.move_scan_to_invalid_pdf_bucket.py` file as the method is only called here.